src/logtab.cpp

```diff
@@ -96,8 +96,34 @@ void LogTab::InitMenus()
     m_hideSelectedType = new QAction(QIcon(GetThemedIcon(":/ctx-hide.png")), "Hide all events of this type", this);
     connect(m_hideSelectedType, &QAction::triggered, this, &LogTab::RowHideSelectedType);
 
+    // Top-level `Export events`
     m_exportToTabAction = new QAction(QIcon(GetThemedIcon(":/ctx-newtab.png")), "Export event(s) to new tab", this);
-    connect(m_exportToTabAction, &QAction::triggered, this, &LogTab::ExportToNewTab);
+    connect(m_exportToTabAction, &QAction::triggered, [this]() { ExportToNewTab(); });
+    // Populate `Export to tab` menu
+    m_exportToTabMenu = new QMenu("Export to new tab",this);
+    m_exportToTabMenu->setIcon(QIcon(GetThemedIcon(":/ctx-newtab.png")));
+    {
+        // This thread
```

Comment:
minor: the comment doesn't match, I guess it is supposed to say "This type"

Reply:
Adjusted here and in the "Highlight all events of this type" code